mlflow/store/artifact/gcs_artifact_repo.py

```diff
@@ -141,3 +148,109 @@ def delete_artifacts(self, artifact_path=None):
         blobs = gcs_bucket.list_blobs(prefix=f"{dest_path}")
         for blob in blobs:
             blob.delete()
+
+    @staticmethod
+    def _validate_support_mpu():
+        if not compare_package_version(
```

Comment:
Does `compare_package_version` throw if `google-cloud-storage` isn't installed?

Reply:
yes but `GCSArtifactRepository` won't be able to be instantiated in this case